auth0_flutter_platform_interface/lib/src/web-auth/web_auth_login_options.dart

```diff
@@ -21,20 +25,28 @@ class WebAuthLoginOptions implements RequestOptions {
       this.invitationUrl,
       this.scheme,
       this.useEphemeralSession = false,
-      this.parameters = const {}});
+      this.parameters = const {},
+      this.safariViewController});
 
   @override
-  Map<String, dynamic> toMap() => {
-        'leeway': idTokenValidationConfig?.leeway,
-        'issuer': idTokenValidationConfig?.issuer,
-        'maxAge': idTokenValidationConfig?.maxAge,
-        'audience': audience,
-        'scopes': scopes.toList(),
-        'redirectUrl': redirectUrl,
-        'organizationId': organizationId,
-        'invitationUrl': invitationUrl,
-        'useEphemeralSession': useEphemeralSession,
-        'parameters': parameters,
-        'scheme': scheme
-      };
+  Map<String, dynamic> toMap() {
+    final map = {
+      'leeway': idTokenValidationConfig?.leeway,
+      'issuer': idTokenValidationConfig?.issuer,
+      'maxAge': idTokenValidationConfig?.maxAge,
+      'audience': audience,
+      'scopes': scopes.toList(),
+      'redirectUrl': redirectUrl,
+      'organizationId': organizationId,
+      'invitationUrl': invitationUrl,
+      'useEphemeralSession': useEphemeralSession,
+      'parameters': parameters,
+      'scheme': scheme,
+      ...safariViewController != null
+          ? {'safariViewController': safariViewController?.toMap()}
```

Comment:
Would it work the same if it always returned the `safariViewController` key, but the value being null if not present?

Reply:
In this case no, as the presence of the key signals that `SFSafariViewController` is to be used (we used a boolean in an earlier iteration but removed it).

Reply:
I've simplified it 👍🏻

Reply:
Added some test coverage for this in https://github.com/auth0/auth0-flutter/pull/198/commits/ab55bc6d40d968d2fc4d84dc5ac4a5606c7ef926